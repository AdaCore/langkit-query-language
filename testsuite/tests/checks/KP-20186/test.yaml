driver: checker
rule_name: kp_20186
project: prj.gpr
