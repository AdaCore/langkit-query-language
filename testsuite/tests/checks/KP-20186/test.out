main.ads:3:9: rule violation: possible occurrence of KP 20186
3 |    type Priv_Range is private;                     --  FLAG
  |         ^^^^^^^^^^

main.ads:5:9: rule violation: possible occurrence of KP 20186
5 |    type Priv_Float_Range is private;               --  FLAG
  |         ^^^^^^^^^^^^^^^^

main.ads:6:9: rule violation: possible occurrence of KP 20186
6 |    type Priv_Enum is private;                      --  FLAG
  |         ^^^^^^^^^

